### PR TITLE
Lodhe Update

### PR DESCRIPTION
**Pull request by Kazkin**
Big code.

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
removed many plushies big sad

File: code/game/objects/random/mob/spider.dm [hunk 1]
```diff
@@ -9,6 +9,7 @@
 				/mob/living/carbon/superior_animal/giant_spider/nurse/midwife = 15,\
 				/mob/living/carbon/superior_animal/giant_spider/nurse/queen = 5,\
 				/mob/living/carbon/superior_animal/giant_spider/nurse/recluse = 4,\
+				/mob/living/carbon/superior_animal/giant_spider/plasma = 4,\
```
Comment (Trilbyspaceclone) [inline code/game/objects/random/mob/spider.dm:12]:
```suggestion
				/mob/living/carbon/superior_animal/giant_spider/plasma = 2,\
```
these should be rarer then recluse as they brake pipes/walls and AOE their own making them a bad breed

File: code/modules/mob/living/carbon/superior_animal/giant_spider/types/hunter.dm [hunk 2]
```diff
@@ -33,6 +33,30 @@
 	melee_damage_lower = 20
 	melee_damage_upper = 25
 
+//bomb spider, very little toxins and not much health but explodes on reaching melee range.
+/mob/living/carbon/superior_animal/giant_spider/plasma
+	name = "plasma spider"
+	desc = "Furry and black, it makes you shudder to look at it. This one is infused with plasma and has mutated spikes poking from its chitin."
```
Comment (ThePainkiller) [inline code/modules/mob/living/carbon/superior_animal/giant_spider/types/hunter.dm:39]:
```suggestion
	desc = "Furry and black, it makes you shudder to look at it. This one is infused with plasma and has mutated spikes protruding from its chitin."
```

File: code/modules/mob/living/carbon/superior_animal/giant_spider/types/hunter.dm [hunk 3]
```diff
@@ -84,3 +108,21 @@
 	..()
 	pixel_x = -16
 	pixel_y = null
+
+/mob/living/carbon/superior_animal/giant_spider/tarantula/emperor/reaper_spider
+	name = "reaper spider"
+	desc = "Furry, white, and black, it makes you shudder to look at it. This one is a massive hulking leviathan capable of striking fear in even the most powerful stalwart of men."
```
Comment (ThePainkiller) [inline code/modules/mob/living/carbon/superior_animal/giant_spider/types/hunter.dm:114]:
```suggestion
	desc = "Furry, white, and black, it makes you shudder to look at it. This one is a massive hulking leviathan capable of striking fear in even the most powerful and stalwart of men."
```

File: code/modules/mob/living/simple_animal/hostile/big.dm [hunk 4]
```diff
@@ -265,3 +282,68 @@
 	..()
 	icon_state = "leaper"
 	icon_living = "leaper"
+
+/mob/living/simple_animal/hostile/hell_pig
+	name = "hell pig"
+	desc = "The venerable evolution of a tengolo charger, morphing into a violent destructive beast hostile to all but its own berserk kind. Hell pigs represent the end of a chargers life \
```
Comment (ThePainkiller) [inline code/modules/mob/living/simple_animal/hostile/big.dm:288]:
```suggestion
	desc = "The venerable evolution of a tengolo charger, morphing into a violent and destructive beast hostile to all but its own berserk kind. Hell pigs represent the end of a charger's life \
```

File: code/modules/mob/living/simple_animal/hostile/big.dm [hunk 5]
```diff
@@ -265,3 +282,68 @@
...
+	as upon reaching a certain age they undergo a metamorphisis that turns them into blood thirsty monsters who rampage until slain."
```
Comment (ThePainkiller) [inline code/modules/mob/living/simple_animal/hostile/big.dm:289]:
```suggestion
	as upon reaching a certain age they undergo a metamorphosis that turns them into bloodthirsty monsters who rampage until slain."
```

File: code/modules/mob/living/simple_animal/hostile/big.dm [hunk 6]
```diff
@@ -265,3 +282,68 @@
...
+	mob_size =  3  // The same as Hivemind Tyrant
+	wander = 1
+	leather_amount = 20
+	bones_amount = 10
+	has_special_parts = TRUE
+	special_parts = list(/obj/item/animal_part/wolf_tooth,/obj/item/animal_part/wolf_tooth)
+
+/mob/living/simple_animal/hostile/hell_pig/wendigo
+	name = "wendigo"
+	desc = "The venerable evolution of a tengolo stalker, morphing into a violent destructive beast hostile to all but its own berserk kind. The wendigo is a violent transformation that involves \
```
Comment (ThePainkiller) [inline code/modules/mob/living/simple_animal/hostile/big.dm:316]:
```suggestion
	desc = "The venerable evolution of a tengolo stalker, morphing into a violent and destructive beast hostile to all but its own berserk kind. The wendigo is a violent transformation that involves \
```

File: code/modules/mob/living/simple_animal/hostile/big.dm [hunk 7]
```diff
@@ -265,3 +282,68 @@
...
+	health = 800
+	move_to_delay = 2
+	turns_per_move = 6
+	speak_emote = list("snarls!", "jerks erratically.")
+	has_special_parts = TRUE
+	special_parts = list(/obj/item/animal_part/wolf_tooth,/obj/item/animal_part/wolf_tooth)
+
+/mob/living/simple_animal/hostile/hell_pig/slepnir
+	name = "slepnir"
+	desc = "The venerable evolution of a tengolo charger, morphing into a violent destructive beast hostile to all but its own berserk kind. The slepnir, unlike its other berserk kin, only attacks \
```
Comment (ThePainkiller) [inline code/modules/mob/living/simple_animal/hostile/big.dm:333]:
```suggestion
	desc = "The venerable evolution of a tengolo charger, morphing into a violent and destructive beast hostile to all but its own berserk kind. The slepnir, unlike its other berserk kin, only attacks \
```

File: code/modules/mob/living/simple_animal/hostile/misc.dm [hunk 8]
```diff
@@ -283,4 +307,35 @@
 	emote_hear = list("gronks.")
 	meat_amount = 3
 	mob_size = 20
-	resistance = 15
\ No newline at end of file
+	resistance = 15
+
+/mob/living/simple_animal/hostile/retaliate/tahca
+	name = "tahca"
+	desc = "A cervine that is, surprisingly, native to the planet that resembles a two headed deer. Despite what what one expect, while docile, it is more than willing to fight if attacked. Hunters value \
```
Comment (ThePainkiller) [inline code/modules/mob/living/simple_animal/hostile/misc.dm:314]:
```suggestion
	desc = "A cervine creature, surprisingly native to this planet, that resembles a two-headed deer. Despite what one would expect, while docile in nature, it is more than willing to fight if attacked. Hunters value \
```

File: code/modules/projectiles/guns/oddity_items.dm [hunk 9]
```diff
@@ -1,11 +1,10 @@
 /obj/item/weapon/gun/projectile/handmade_pistol/reliable
 	name = "\"Reliable\" handmade pistol"
 	desc = "An anomalous weapon created by an unknown person (or group?), their work marked by a blue cross, these weapons are known to vanish and reappear when left alone. \
-	Whoever made this particular model did exceptional work, unlike lesser handmade pistols this one jams less, holds three bullets, and has much better shot capability. Uses .35 caliber."
+	Whoever made this particular model did exceptional work, unlike lesser handmade pistols this one jams less and has much better shot capability. Uses .35 caliber."
```
Comment (ThePainkiller) [inline code/modules/projectiles/guns/oddity_items.dm:4]:
```suggestion
	Whoever made this particular model did exceptional work. Unlike lesser handmade pistols, this one jams less and has much better shot capability. Uses .35 caliber."
```

File: code/modules/projectiles/guns/projectile/shotgun/hunter_crossbow.dm [hunk 10]
```diff
@@ -0,0 +1,31 @@
+/obj/item/weapon/gun/projectile/shotgun/pump/hunter_crossbow
+	name = "bolt lance"
+	desc = "A modified crossbow powered by an internal battery device that rapidly winds up a hydraulic winch to deliver a devistating superheated bolt shot. Hand-made by the local hunting lodge it seems, using a design commonly found on the homeworld of the Sablekyne."
```
Comment (ThePainkiller) [inline code/modules/projectiles/guns/projectile/shotgun/hunter_crossbow.dm:3]:
```suggestion
	desc = "A modified crossbow powered by an internal battery device that rapidly winds up a hydraulic winch to deliver a devastating, superheated bolt shot. Hand-crafted by the local Hunting Lodge, it uses a design commonly found on Onkarth, the homeworld of the Sablekyne."
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]
Just a couple grammar and orthographic corrections...

File: code/modules/mob/living/simple_animal/hostile/misc.dm (hunk 8 above)
Comment (ThePainkiller) [reply code/modules/mob/living/simple_animal/hostile/misc.dm:314]: This one is still stretching it. _"Despite what one would expect"_ is almost unnecessary given common sense and cohesion (_"docile in nature"_ already tells you that the creature is tame by default, and the _"while"_ at the start is already foreboding of an exception to the rule about to get mentioned) and could be taken out outright without changing the spirit of the flavor text.

File: code/game/objects/random/mob/spider.dm (hunk 1 above)
Comment (Kazkin) [reply code/game/objects/random/mob/spider.dm:12]: It gives the guild something to repair.

**Pull request merged**